packages/fx-core/templates/bicep/azureWebApp.config.module.bicep

```diff
@@ -40,12 +41,20 @@ resource webAppSettings 'Microsoft.Web/sites/config@2021-02-01' = {
   name: '${webAppName}/appsettings'
   properties: union({
     {{#if (contains "aad-app" connections)}}
-    INITIATE_LOGIN_ENDPOINT: uri(provisionOutputs.azureWebApp{{scenario}}Output.value.siteEndpoint, 'auth-start.html') // The page is used to let users consent required OAuth permissions during bot SSO process
+      {{#if (contains "teams-tab" connections)}}
+    TAB_APP_ENDPOINT: webappEndpoint
+    TeamsFx__Authentication__ClientId: m365ClientId // Client id of AAD application
+    TeamsFx__Authentication__ClientSecret: m365ClientSecret // Client secret of AAD application
+    TeamsFx__Authentication__InitiateLoginEndpoint: uri(webappEndpoint, 'auth-start.html') // The page is used to let users consent required OAuth permissions during bot SSO process
```

Comment:
Nit: 
1. Make a common variable above all - `initiateLoginEndpoint = uri(provisionOutputs.azureWebApp{{scenario}}Output.value.siteEndpoint, 'auth-start.html') ` and reference it in below. 
2. The indentation can be better.
3. `INITIATE` is a verb, `initial` may be a better naming.

Reply:
2. The indentation of template looks wired, it's because we must make sure the final output bicep's indentation perfect.
3. You are right, but the variable name is determined by SDK, we can't change it.